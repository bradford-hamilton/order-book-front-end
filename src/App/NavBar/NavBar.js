import React from 'react';
import { NavItem, Navbar, Nav } from 'react-bootstrap';

const NavBar = () => (
  <Navbar inverse>
    <Navbar.Header>
      <Navbar.Brand>
        <a href="/">MasterBook</a>
      </Navbar.Brand>
    </Navbar.Header>
  </Navbar>
);

export default NavBar;
