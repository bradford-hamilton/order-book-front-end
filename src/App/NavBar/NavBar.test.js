import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import NavBar from './NavBar';

configure({ adapter: new Adapter() });

function setup() {
  const comp = shallow(<NavBar />);
  return { comp };
}

describe('<NavBar />', () => {
  it('renders without crashing', () => {
    const { comp } = setup();
    expect(comp).toBeDefined();
  });

  test('snapshot', () => {
    const tree = renderer.create(<NavBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
