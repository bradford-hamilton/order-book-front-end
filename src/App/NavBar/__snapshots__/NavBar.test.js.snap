// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> snapshot 1`] = `
<nav
  className="navbar navbar-inverse"
>
  <div
    className="container"
  >
    <div
      className="navbar-header"
    >
      <a
        className="navbar-brand"
        href="/"
      >
        MasterBook
      </a>
    </div>
  </div>
</nav>
`;
